quadriga/src/main/java/edu/asu/diging/quadriga/core/service/EventGraphService.java

```diff
@@ -32,6 +34,18 @@ public interface EventGraphService {
      * @return total count
      */
     public long getNumberOfSubmittedNetworks(ObjectId collectionId);
+
+    /**
+     * Returns all event graphs by collection id
+     * @param collectionId
+     * @param size 
+     * @param page 
+     * @return
+     */
```

Comment:
why does this method get javadoc but not the other?

Reply:
All methods now have javadoc.

Reply:
also there is some text missing (e.g. explanation of parameters)